### PR TITLE
Volcano altitude above sea level / above vent

### PR DESCRIPTION
**Pull request by babrodtk**
This PR makes it clear if user supplies altitude above vent or above altitude. 

Also changes logging to screen so that it is easier to spot mistakes when submitting a job. The user now sees the parameters that are used to generate the emissions.

### CODE REVIEW

**Comment by babrodtk** [comment]
This has been tested using vgl-launch, and appears to perform as intended.

**Comment by babrodtk** [comment]
@heikoklein Can you take a look at this? The changes are so significant that I'd like a "fresh" pair of eyes to check that I haven't introduced obvious bugs.

File: utils/SnapPy/Snappy/EEMEP/Controller.py [hunk 1]
```diff
@@ -91,20 +91,24 @@ def __init__(self):
         self.lastQDict = {}
         self.lastLog = ""
 
-    def write_log(self, txt:str):
-        if (txt != self.lastLog):
-            self.lastLog = txt
-            debug(txt)
-            self.main.evaluate_javaScript('updateEemepLog({0});'.format(json.dumps(txt)))
+    def write_log(self, txt:str, clear_log=False):
```
Comment (babrodtk) [inline utils/SnapPy/Snappy/EEMEP/Controller.py]: The clear_log flag is used to clear the log explicitly. Otherwise, it keeps the log from previously.

**Comment by heikoklein** [comment]
Functionality with height looks good. bottom always 0 in volcano.xml? Not sure if bottom is used by emep...

On screen log gets too long: 100 messages like:
Queue busy 2020-02-03 08:52:36
working pid: 163118 on node: xvis-m4b
current volcano: '/lustre/storeB/project/fou/kl/eva/eemep/runs/reykjanes/volcano.xml'
  started by: emepgrp (emepgrp)
working pid: 163118 on node: xvis-m4b
current volcano: '/lustre/storeB/project/fou/kl/eva/eemep/runs/reykjanes/volcano.xml'
  started by: emepgrp (emepgrp)

Should be cleared, and log-window should never become larger than 30-40 lines.


Some minor textual changes in feedback/log:
Please check ash cloud datum. -> Please check ash cloud. (No datum mentioned in GUI)

Ash cloud height measured from vent: 0.00 km, rate: 0 kg/s, vent height: 2.01 -> 
  volcano height: 2.01 km (unit + vent/volcano change)

**Comment by babrodtk** [comment]
I have discovered some race conditions in the logging/running whilst working with this, but have not addressed all of the issues in this PR.

**Pull request merged**